docs/ci-integration/pull-through-cache.md

```diff
@@ -0,0 +1,40 @@
+# Pull Through Cache
+
+## Introduction
+
+Docker Hub, Quay, and other registry providers all have pull limits, and costs associated with running them. Running large builds (or many small builds, frequently) may incur excess costs, rate limiting, or both. This guide will help you set up your own "pull-through" cache to combat the limitations imposed by registry providers.
```

Comment:
`combat the limitations imposed by registry providers` - I would reword this as "optimize network traffic". Feels less like a battle with any of the provider :-)

Reply:
Worded a little different but dropped `combat`.